src/telliot_feeds/reporters/rng_interval.py

```diff
@@ -311,6 +312,9 @@ async def report(self) -> None:
         """Submit latest values to the TellorFlex oracle."""
         logger.info(f"RNG reporting interval: {INTERVAL} seconds")
         while True:
+            online = await is_online()
+            if not online:
+                logger.warning("Unable to connect to the internet!")
             _, _ = await self.report_once()
```

Comment:
This one tries to report even if no internet

Reply:
true, good catch